src/graph/on_disk_graph.cpp

```diff
@@ -30,11 +31,15 @@ namespace graph {
 static std::unique_ptr<RelTableScanState> getRelScanState(MemoryManager& mm,
     const TableCatalogEntry& relEntry, const RelTable& table, RelDataDirection direction,
     ValueVector* srcVector, ValueVector* dstVector, ValueVector* relIDVector,
-    expression_vector properties, const Schema& schema, const ResultSet& resultSet) {
+    expression_vector predicateEdgeProperties, std::optional<column_id_t> edgePropertyID,
```

Comment:
This is fine for this PR.

@acquamarin ultimately u should just pass `edgeProperties` from front-end to replace `predicateEdgeProperties & edgePropertyID`